src/08_comprehensions.py

```diff
@@ -26,7 +26,7 @@
 
 a = ["foo", "bar", "baz"]
 
-y = []
+y = [x.upper() for x in a]
```

Comment:
Maybe consider using more explicit naming convention for better readability.

example: y = [word.upper() for word in a]